### PR TITLE
Update sample5.vb

### PR DESCRIPTION
**Pull request by 6eh01der**
Correct VBS code (current is not VBS, but JavaScript).

### CODE REVIEW

**Comment by 6eh01der** [comment]
@microsoft-github-policy-service agree

File: iis/configuration/moduleProviders/index/samples/sample5.vb [hunk 1]
```diff
@@ -1,12 +1,12 @@
-var adminManager = new ActiveXObject("Microsoft.ApplicationHost.WritableAdminManager"); 
-adminManager.CommitPath = "MACHINE/WEBROOT"; 
-adminManager.SetMetadata("pathMapper", "AdministrationConfig");
-var moduleProvidersSection = adminManager.GetAdminSection("moduleProviders", "MACHINE/WEBROOT"); 
+Set adminManager = WScript.CreateObject("Microsoft.ApplicationHost.WritableAdminManager")
+adminManager.CommitPath = "MACHINE/WEBROOT"
+adminManager.SetMetadata "pathMapper", "AdministrationConfig"
+Set modulesSection = adminManager.GetAdminSection("modules", "MACHINE/WEBROOT")
```
Comment (wadepickett) [inline iis/configuration/moduleProviders/index/samples/sample5.vb:4]:
```suggestion
Set moduleProvidersCollection = adminManager.GetAdminSection("modules", "MACHINE/WEBROOT")
```

Comment (wadepickett) [reply iis/configuration/moduleProviders/index/samples/sample5.vb:4]: `moduleProvidersCollection` was changed to `modulesSection` but then that change was not made through the rest of the new VB code.  Setting it back to `moduleProvidersCollection` so it works with the rest of the code and compiles.

**Review by wadepickett — APPROVED** [review]
Approving with a small fix.

**Comment by wadepickett** [comment]
Thanks @6eh01der!  Greatly appreciated you spotted this issue and provided a VB fix!

**Comment by wadepickett** [comment]
Forced refresh build close/re-open

**Comment by wadepickett** [comment]
Just realized I added an error, my apologies, fixing...

File: iis/configuration/moduleProviders/index/samples/sample5.vb [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+adminManager.CommitPath = "MACHINE/WEBROOT"
+adminManager.SetMetadata "pathMapper", "AdministrationConfig"
+Set moduleProvidersCollection = adminManager.GetAdminSection("modules", "MACHINE/WEBROOT")
 
-var moduleProvidersCollection = moduleProvidersSection.Collection;
-var addElement = moduleProvidersCollection.CreateNewElement("add");
-addElement.Properties.Item("name").Value = "ContosoProvider";
-addElement.Properties.Item("type").Value = "Contoso.Provider, System.Web, Version=1.0.0.0, Culture=neutral, PublicKeyToken=426f62526f636b73";
-moduleProvidersCollection.AddElement(addElement);
+Set moduleProvidersCollection = moduleProvidersCollection.Collection
```
Comment (wadepickett) [inline iis/configuration/moduleProviders/index/samples/sample5.vb:6]:
```suggestion
Set moduleProvidersCollection = modulesSection.Collection
```

File: iis/configuration/moduleProviders/index/samples/sample5.vb [hunk 3]
```diff
@@ -1,12 +1,12 @@
-var adminManager = new ActiveXObject("Microsoft.ApplicationHost.WritableAdminManager"); 
-adminManager.CommitPath = "MACHINE/WEBROOT"; 
-adminManager.SetMetadata("pathMapper", "AdministrationConfig");
-var moduleProvidersSection = adminManager.GetAdminSection("moduleProviders", "MACHINE/WEBROOT"); 
+Set adminManager = WScript.CreateObject("Microsoft.ApplicationHost.WritableAdminManager")
+adminManager.CommitPath = "MACHINE/WEBROOT"
+adminManager.SetMetadata "pathMapper", "AdministrationConfig"
+Set moduleProvidersCollection = adminManager.GetAdminSection("modules", "MACHINE/WEBROOT")
```
Comment (wadepickett) [inline iis/configuration/moduleProviders/index/samples/sample5.vb:4]:
```suggestion
Set modulesSection = adminManager.GetAdminSection("modules", "MACHINE/WEBROOT")
```

**Comment by wadepickett** [comment]
I needed to change the filename to a .vbs extension, it was left as .vb and the doc that refers to it therefore needs a link update.

**Comment by wadepickett** [comment]
Verified in [Internal review doc build}(https://review.learn.microsoft.com/en-us/iis/configuration/moduleproviders/?branch=pr-en-us-956)

**Pull request merged**